src/components/Panel/Rightside/Members.vue

```diff
@@ -91,31 +92,64 @@ export default {
 
   data () {
     return {
-      userQuery: null,
+      query: null,
+
+      members: [],
+      nonMembers: [],
     }
   },
 
   computed: {
     ...mapGetters({
       users: 'users/list',
     }),
+  },
 
-    members () {
-      return this.users.filter(u => this.channel.isMember(u.userID))
+  watch: {
+    'query' (query) {
+      if (query) {
+        const nonMembers = []
+        const limit = 15
+        this.$SystemAPI.userList(({ query, limit })).then(({ set = [] }) => {
+          set.forEach(u => {
+            if (!this.channel.isMember(u.userID) && u.userID !== this.$auth.user.userID) {
+              nonMembers.push(new User(u))
+            }
+          })
+          this.nonMembers = nonMembers
+        })
+      } else {
+        this.nonMembers = []
+      }
```

Comment:
We should probably debounce or throttle this; don't want to make API sweet too much